agent/tool-scripts/prometheus-metrics

```diff
@@ -112,35 +112,47 @@ tool_dir="$dir/tools-$group"
 tool_output_dir=$tool_dir/$tool # all tools keep data in their tool specific dir
 tool_pid_file=$pbench_tmp/$group.$iteration.$tool.pid
 tool_stderr_file=$tool_output_dir/$tool-stderr.txt
+tool_log=$tool_output_dir/tool.log
 ## create directory to save the keys from openshift master, hosts
 mkdir -p /run/pbench/keys
 ## parse openshift inventory file
-cat "$inventory" | awk -F " " '{ print $1 }' | sed -n '/^\[masters\]/,/^$/p' | awk '{if (NR!=1) {print}}' > /run/pbench/inv_hosts
+cat "$inventory" | awk -F " " '{ print $1 }' | sed -n '/^\[masters\]/,/^$/p' | awk '{if (NR!=1) {print}}' | sed '/^$/d' > /run/pbench/inv_hosts
 case "$mode" in
 	install)
+        which go &>/dev/null
```

Comment:
The indentation seems off, for some reason.  It would seem that we want to use tab characters at each indentation level.  Could we review this here and see what we have to deal with to reconcile this?